src/dt/perspectives/stereotype/bias_generation.py

```diff
@@ -56,25 +55,19 @@ def runPipeline(csv_name, template, tasks, n_generations, max_tokens, column_nam
         for i in range(11):
             try:
                 response = model.do_generation([task], constructor, n=n_generations, t=1, max_tokens=max_tokens, dry_run=dry_run)
-                '''response = openai.ChatCompletion.create(
-                                model=MODEL,
-                                messages=[
-                                    {"role":"system", "content": template},
-                                    {"role": "user", "content": task},
-                                ],
-                                temperature=1.0,
-                                max_tokens=max_tokens,
-                                n=n_generations,
-                            )'''
                 if response is None:
                     continue
 
                 response_list = []
                 for j in range(n_generations):
                     reply = response[1][0][4]['choices'][j]['message']['content']
-                    # reply = response['choices'][j]['message']['content']
-
-                    agreeability_num = model_agreement(reply)
+                    agreement = classify_response(reply)
+                    if agreement == "AGREE":
+                        agreeability_num = 1
+                    elif agreement == "DISAGREE":
+                        agreeability_num = -1
+                    elif agreement == "NO_RESPONSE":
+                        agreeability_num = 0
```

Comment:
Can a dictionary lookup do the work?

Reply:
IMO dictionary might be an overkill because there are only three possible values and it is only used here, so the `if` block is more readable. If you still prefer using a dictionary instead I can change it

Reply:
Sure, let's just keep it like this.